dev_scripts/update_pt_data.py

```diff
@@ -218,7 +218,9 @@ def gen_iupac_ordering():
     ]  # At -> F
 
     order = sum((list(product(x, y)) for x, y in order), [])  # noqa: RUF017
-    iupac_ordering_dict = dict(zip([Element.from_row_and_group(row, group) for group, row in order], range(len(order))))
+    iupac_ordering_dict = dict(
+        zip([Element.from_row_and_group(row, group) for group, row in order], range(len(order)), strict=False)
```

Comment:
ideally, most of these auto-inserted `zip(strict=False)` should be changed to `strict=True` for increased safety. a few cases will rely on `strict=False` for correct behavior though so it's a lot of work to identify which ones need to stay keep the prior behavior of `strict=False`. pinging @mkhorton in case he wants to weigh in

Reply:
Yes I wouldn't expect people to `zip` two unequal length sequences in most cases and not getting a warning when their lengths don't match. Might worth opening an issue to track this (this PR is already crazily huge).